.env

```diff
@@ -0,0 +1 @@
+INSTALL_PREFIX="/var/repos/mirror-kickstarter"
```

Comment:
`.env` I think it should be the optional file.

I think it should create the `.env.example` file and let the `.env` file be under the `.gitignore` file.

And add some `.env` setting description in the `README.md`.